mojave/modules/crafting/recipes/recipes.dm

```diff
@@ -247,3 +247,75 @@
 				/obj/item/stack/sheet/ms13/plank = 1)
 	category = CAT_SMELTER
 	crafting_interface = CRAFTING_BENCH_SMELTER*/
+
+/datum/crafting_recipe/smelt_copper
+	name = "refine copper ore"
+	result = /obj/item/stack/sheet/ms13/refined_copper/two
+	time = 3 SECONDS
```

Comment:
These should probably have a hammer tool requirement since you're presumably hammering out a couple ingots.